README.md

```diff
@@ -119,3 +120,6 @@
 [#53432]: https://github.com/rust-lang/rust/issues/53432
 [21]: https://play.rust-lang.org/?gist=448e4a6646e0048b027b03f607a612dc&version=nightly&mode=debug&edition=2015
 [22]: https://play.rust-lang.org/?gist=b61db3de15c2c0716d76d7e5df9abb02&version=nightly&mode=debug&edition=2015
+[23]: https://play.rust-lang.org/?gist=2869a3fead3981815e077700d68de8eb&version=nightly&mode=debug&edition=2015
+[24]: https://play.rust-lang.org/?gist=d404421b51edcb0c2d6fb700a313be95&version=nightly&mode=debug&edition=2015
```

Comment:
confirm, error looks correct, though it is not rust-lang/rust#52059 — I'm not sure what AST bug this is. Looks like something to do with `DerefMut`...?